### PR TITLE
docs: updating quickstart to account kit re-branding

### PR DESCRIPTION
**Pull request by avarobinson**
Updating quickstart copy to include social auth and rebranding of account kit. updating images to include account kit. 

# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

---

## PR-Codex overview
This PR focuses on updating documentation and images related to authentication processes using UI components and React hooks in the `Account Kit`. It enhances clarity and organization, particularly for email and social authentication setups.

### Detailed summary
- Renamed sections to emphasize "Authentication" instead of "Using".
- Updated instructions for API key retrieval and account configuration.
- Improved clarity in email and social authentication setup steps.
- Removed outdated images and added new ones for better guidance.
- Enhanced layout and styling for better readability in the `quickstart` guide.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Review by moldy530 — APPROVED** [review]

File: site/pages/react/ui-components.mdx [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Authentication with UI componenets
 description: How to use our pre-built authentication component in your React app
 ---
 
-# Using our UI components
+# Authentication with our UI componenets
```
Comment (dphilipson) [inline site/pages/react/ui-components.mdx:6]: Typo: "componenets" -> "components"

File: site/shared/get-api-key.mdx [hunk 2]
```diff
@@ -1,24 +1,52 @@
...
+          style={{ width: "50%" }}
+        />
+
+        <br />
+        **Social auth**
+
+        If you want to enable social login, toggle which auth providers
+        you want to support.
+
+        - Add [http://localhost:3000](https://localhost:3000/) as a "whitelisted origin" for testing.
```
Comment (dphilipson) [inline site/shared/get-api-key.mdx:38]: The second url in this line should be http not https.

File: site/shared/get-api-key.mdx [hunk 3]
```diff
@@ -1,24 +1,52 @@
...
+        - Optionally enter your own Oauth credentials or use our defaults.
```
Comment (dphilipson) [inline site/shared/get-api-key.mdx:39]: Let's capitalize as "OAuth" in documentation.

**Review by dphilipson — DISMISSED** [review]
Looks good! Noted some typos.

**Review by dphilipson — APPROVED** [review]

**Pull request merged**